core/src/services/persy/backend.rs

```diff
@@ -175,7 +176,12 @@ impl kv::Adapter for Adapter {
     }
 
     async fn get(&self, path: &str) -> Result<Option<Vec<u8>>> {
-        self.blocking_get(path)
+        let cloned_self = self.clone();
+        let cloned_path = path.to_string();
+        task::spawn_blocking(move || cloned_self.blocking_get(cloned_path.as_str()))
```

Comment:
From my perspective, This may make us have to rely on tokio, and it may be difficult to support other asynchronous runtimes.

Reply:
Good catch, I think we need to make a choice between portable and performance. Or we need to abstract a layer to adapt the different async runtimes.

off-topic, would you mind introducing to me another popular rust async runtime except for tokio?

Reply:
cc @Xuanwo

Reply:
OpenDAL is tokio only for now. I think it's ok to add tokio::task here and decide how to fix it when we are trying to support other runtimes.

Reply:
> I think it's ok to add tokio::task here and decide how to fix it when we are trying to support other runtimes.

I'd rather move on than wait for a *perfect* solution, unless the drawbacks are unaffordable.